### DISCUSSION TITLE
new user can not create first workspace

### DISCUSSION THREAD

**Discussion by illuminatrix8** [AntlerVC/firetable#848] [Support / Q&A] [answered]
Steps
- register as a new user
- sees message to "+ create workspace"
- fill in 2 steps of onboarding popup in name, row, team size, and usecase
- spinner loads, page refreshes, and loads 2 steps popup again.

**Comment by notsidney** [comment]
Hi @illuminatrix8, just checking that this has been resolved now?

**Comment by illuminatrix8** [comment]
Hi Sidney,

Still same problem

**Comment by harinij** [comment, marked as answer]
Hi @illuminatrix8 most of these issues are resolved in our recent releases. If you are still facing any issue it could be related to the specific project, if you can email us with the email used in the setup/projectId, we can take a look and get it resolved → support@rowy.io
